bundles/org.eclipse.jdt.doc.user/reference/preferences/run-debug/ref-console_ansi.htm

```diff
@@ -0,0 +1,185 @@
+  <head>
+    <meta name="copyright" content=
+      "Copyright (c) IBM Corporation and others 2022. This page is made available under license. For full details see the LEGAL in the documentation book that contains this page." />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="Content-Style-Type" content="text/css" />
+    <link rel="stylesheet" href="../../../book.css" charset="ISO-8859-1" type="text/css" />
+    <title>
+      Ansi Support Preferences
+    </title>
+    <script language="JavaScript" src=
+      "PLUGINS_ROOT/org.eclipse.help/livehelp.js" type="text/javascript">
+    </script>
+  </head>
+  <body>
+    <h1>Ansi Support Preferences</h1>
+
+    <p>The following preferences can be set using the <a class="command-link" href=
+        'javascript:executeCommand("org.eclipse.ui.window.preferences(preferencePageId=org.eclipse.ui.internal.console.ansi.preferences.AnsiConsolePreferencePage)")'>
+      <img src="PLUGINS_ROOT/org.eclipse.help/command_link.svg" alt=
+        "Opens the Ansi Support preference page" /><b>Run/Debug &gt; Console &gt; Ansi Support</b></a> preference page.</p>
+    <p>Many console applications can interpret ANSI sequences to produce styled output.<br>
+      The codes supported by these applications is not consistent, but the most common attributes
+      supported include foreground and background colors, bold, italic, underline, invert.</p>
+    <p>See <a href="https://en.wikipedia.org/wiki/ANSI_escape_code">Wikipedia: ANSI escape code</a> for details.</p>
+    <p>The Eclipse Console supports 16 colors / 256 colors / true-colors for foreground and background, color palettes,
+      and attributes like bold, italic, underline, invert, conceal, strike, crossed-out, framed.</p>
+    <p>Only “CSI n m” (<a href="https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_(Select_Graphic_Rendition)_parameters">SGR,
+      Select Graphic Rendition</a>) control sequences are supported.<br>
+      They set display attributes, but cannot be used to change the content of the output
+      (erase actions, cursor positioning, scrolling).<br>
+      Non-SGR sequences are not interpreted, but are recognized and hidden, so you will not see them.</p>
+    <p>This functionality is available for all the text output to the standard output or standard error.
+      It works for Java, C/C++, Perl, Python, shell, Groovy, Maven, any Eclipse supported language.</p>
+
+    <table border="1" cellspacing="0" cellpadding="5">
+      <thead>
+        <tr>
+          <th rowspan="1" colspan="1">
+            <p>Option</p>
+          </th>
+          <th rowspan="1" colspan="1">
+            <p>Description</p>
+          </th>
+          <th rowspan="1" colspan="1">
+            <p>Default</p>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td rowspan="1" colspan="1" valign="top" align="left">
+            <p>Plugin enabled</p>
```

Comment:
Please update preference label according to the current page state

Reply:
Done.